libensemble/gen_funcs/persistent_tasmanian.py

```diff
@@ -10,10 +10,10 @@
 from libensemble.tools import parse_args
 from libensemble.tools.persistent_support import PersistentSupport
 
-__all__ = [
```

Comment:
I dont think we should remove this- doesn't the docs read this

Reply:
I had to verify this when I was puzzling why that line wasn't covered before, but it turns out `__all__` only gets covered when one does:

`from some_module import *`

This isn't done in our code, or the docs:

```
.. automodule:: persistent_tasmanian
  :members: sparse_grid_batched, sparse_grid_async
  :undoc-members:
```